ethsigner/src/main/java/tech/pegasys/ethsigner/jsonrpc/response/JsonRpcError.java

```diff
@@ -88,7 +88,10 @@
   // Private transaction errors
   ENCLAVE_IS_DOWN(-50100, "Enclave is down"),
   UNIMPLEMENTED_PRIVATE_TRANSACTION_TYPE(-50100, "Unimplemented private transaction type"),
-  PRIVATE_TRANSACTION_RECEIPT_ERROR(-50100, "Error generating the private transaction receipt");
+  PRIVATE_TRANSACTION_RECEIPT_ERROR(-50100, "Error generating the private transaction receipt"),
```

Comment:
Should this be in this PR? (timeout is good, PrivTransaction ... not so much maybe.)

Reply:
That was already there, didn't add it. Thinking it's showing up as a change because I changed the semicolon on the end of line to to a comma to add timeout error.